lib/package.js

```diff
@@ -595,9 +599,9 @@ exports.processPackage = function(pkg, dir, pjson, postload, isCDN) {
         curDeps = pjson.dependencies && Object.keys(pjson.dependencies) || [];
         return endpoint.build(pjson, dir);
       })
-      .catch(function(e) {
+      .catch(function() {
         throw 'Error building package `' + pkg.name + '`.';
```

Comment:
The endpoint does its own error reporting already - https://github.com/jspm/jspm-cli/blob/master/lib/endpoint.js#L161.